onnxruntime/core/providers/cpu/math/matmul_integer.cc

```diff
@@ -1,17 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include "matmul_quant_prepack_base.h"
+
 #include "core/framework/op_kernel.h"
 #include "core/providers/cpu/math/matmul_helper.h"
 #include "core/util/math_cpuonly.h"
 #include "core/util/qmath.h"
+#include "core/mlas/inc/mlas.h"
```

Comment:
clean up the headers here? mlas.h isn't needed as it's in the base class along with op_kernel.h